packages/kernel-5.10/config-bottlerocket

```diff
@@ -8,47 +8,6 @@ CONFIG_EXT4_FS=y
 CONFIG_BLK_DEV_NVME=y
 CONFIG_NVME_CORE=y
 
-# SATA support
-CONFIG_BLK_DEV_SD=y
-CONFIG_SATA_AHCI=y
-CONFIG_ATA=y
-CONFIG_ATA_PIIX=y
-
-# Network support
-CONFIG_ETHERNET=y
-CONFIG_NET_CORE=y
-CONFIG_NETDEVICES=y
```

Comment:
Are these actually necessary to define? They all seem pretty essential. It might be better to drop them.

Reply:
Right, we can drop these. They are selected anyways.